stdlib/builtins.pyi

```diff
@@ -118,16 +118,16 @@ class object:
 class staticmethod(object):  # Special, only valid as a decorator.
     __func__: Callable[..., Any]
     __isabstractmethod__: bool
-    def __init__(self, f: Callable[..., Any]) -> None: ...
+    def __init__(self, __f: Callable[..., Any]) -> None: ...
     def __new__(cls: Type[_T], *args: Any, **kwargs: Any) -> _T: ...
```

Comment:
What's with the way that staticmethod and classmethod have unmatching signatures for __new__ and __init__?